src/templates/Footer/__tests__/footer.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import Footer from 'templates/Footer';
+
+/* eslint-disable */
+jest.mock('atoms/Icon', () => {
```

Comment:
This mock is going to be pretty common. Do we want to add it to the template, but commented out?